app/views/backend/product_nature_variants/show.html.haml

```diff
@@ -24,6 +24,31 @@
         %span.value
           %i.picto{class: "picto-stock"}
           = resource.current_stock.to_s + " " + resource.unit_name
+    - if resource.current_stock == 0
+      .med-info.important
+        %span.title CUMP
```

Comment:
Every human word must i18nized. You can use `AR.human_attribute_name` or `.tl` methods to localize these titles